pyvrp/plotting/plot_objectives.py

```diff
@@ -58,6 +58,10 @@ def _plot(x, y, *args, **kwargs):
     y = [d.avg_cost for d in result.stats.feas_stats]
     _plot(x, y, label="Feas. avg.", c="tab:green", alpha=0.3, ls="--")
 
+    # Use best objectives to set reasonable y-limits
+    best = np.nanmin([d.best_cost for d in result.stats.feas_stats])
+    ax.set_ylim(best * 0.95, best * 1.15)
```

Comment:
Can we make `ylim_adjust` a tuple argument with defaults `(0.95, 1.15)`?

Reply:
I can imagine this is really run-dependent, so 0.95 and 1.15 should work in most cases, but sometimes they might not. It'd be nice to have some control on those cases.